openapi/ga4gh-tool-discovery.yaml

```diff
@@ -585,6 +554,13 @@ definitions:
       - url
       - id
     properties:
+      author:
+        type: array
+        items:
+          type: string
+        description: >-
+          Contact information for the author of this tool entry in the registry.
```

Comment:
Since it's now on the version, should it say something like `...the author of this iteration of the tool entry...` or `...the author of this version of the tool entry...`? Or maybe it's redundant. Throwing it out there for your consideration.

This comment applies to all the of changes.